docs/modules/ROOT/pages/how-tos/new_minor.adoc

```diff
@@ -0,0 +1,50 @@
+= Checklist to prepare for a new minor version of OpenShift 4
+
+. Read and summarize release notes
++
+Read the release notes.
+Look for things that affect how we use OpenShift.
+Ensure each change which impacts us gets addressed.
++
+Write a summary and add it to xref:oc4:ROOT:references/release_notes.adoc[our release notes summary].
```

Comment:
And remove OpenShift versions which we don't support anymore (usually current version - 1, but I suggest keeping current version - 2)